brainstormforce/astra
# feat: Customizer redesign for Astra v3.1.0

### Description
- Customizer redesign includes following parameters
- Major improvements
1. Slider input control revamped with WordPress's core range control.
2. Color-group implemented for better UX.
3. Select group converted into multi-button group which leads better UX.
4. Checkbox control revamped with WordPress's toggle control.
5. Align selection field also revamped with mutli-button control for better UX.
- Design improvements -
1. Color picker UI changes
2. Spacing control design updated.
3. Implemented divider in between all controls so controls are separated properly.
4. Drag-drop meta structure control design's also revamped.

### Types of changes
- Customizer control revamped
- CSS changed as per design perspective for customizer

### How has this been tested?
- With normal controls & controls inside group control
- Test customizer-redesign with Old header footer
- Check with addon/theme only updated
- Verify if previous values comes as it is in new controls after update (majorly in following controls)
1. Checkbox -> Toggle control
2. Slider -> Rangecontrol
3. Setting (color) group -> Color-group
- Cross-browser testing
- RTL language testing

### Checklist:
